src/typed-error.ts

```diff
@@ -1,42 +1,43 @@
 /// <reference path='ext.d.ts'/>
+
```

Comment:
The beauty is the matter of taste, the whole idea of prettier is to have some readable and consistent style and stop bikeshedding altogether, without any manual changes at the same time.
I really think we should start using it, and the tool will become better and better over time (as of now I'd personally prefer couple of places to be different, but it's such a minor thing compared to the convenience of not thinking about the style altogether)